core-graphics-types/src/lib.rs

```diff
@@ -8,7 +8,8 @@
 // except according to those terms.
 
 extern crate core_foundation;
-extern crate libc;
+#[cfg(feature = "objc2")]
```

Comment:
I think there's value in having a standalone PR that adds the `Encode` traits for `objc2` to `core-graphics-types` via a feature. Would that help for some of what you maintain in the `objc2` family of crates?

It is separate from converting the `cocoa` and `cocoa-foundation` crates, right?

Reply:
Hmm, I _think_ it would help, though I'll have to mull it over; it kinda depends on how long it will take me to get to doing https://github.com/madsmtm/objc2/issues/556 (in which case all the types in this repo will just be autogenerated), and I suspect a lot of people would be annoyed at the extra dependency.